central/complianceoperator/v2/compliancemanager/manager_impl.go

```diff
@@ -175,7 +178,15 @@ func (m *managerImpl) ProcessScanRequest(ctx context.Context, scanRequest *stora
 		}
 
 		// Update status in DB
-		err = m.scanSettingDS.UpdateClusterStatus(ctx, scanRequest.GetId(), clusterID, status)
+		cluster, exists, err := m.clusterDS.GetCluster(ctx, clusterID)
+		if err != nil {
+			return nil, err
+		}
+		if !exists {
+			return nil, errors.Errorf("could not pull config for cluster %q because it does not exist", clusterID)
+		}
+
+		err = m.scanSettingDS.UpdateClusterStatus(ctx, scanRequest.GetId(), clusterID, status, cluster.GetName())
```

Comment:
```suggestion
		clusterName, exists, err := m.clusterDS.GetClusterName(ctx, clusterID)
		if err != nil {
			return nil, err
		}
		if !exists {
			return nil, errors.Errorf("could not pull config for cluster %q because it does not exist", clusterID)
		}

		err = m.scanSettingDS.UpdateClusterStatus(ctx, scanRequest.GetId(), clusterID, status, clusterName)
```